tests/testthat/test-visr_survfit.R

```diff
@@ -510,4 +512,20 @@ testthat::test_that("T5.2 The final object is a ggplot of class `ggsurvfit`.", {
   testthat::expect_true(inherits(survfit_plot, "ggsurvfit"))
 })
 
+# Requirement T6 ---------------------------------------------------------------
+
+testthat::context("visr_plot - T6. The final object does not exclude parts of KM estimate.")
+
+testthat::test_that("T6.1 The final object zooms and does not exclude trialing pieces of lines.", {
```

Comment:
I'm not a big fan of creating the actual and test plot with similar code. We should update this throughout the package. Could we somehow get the line information and ensure it is displayed?

Reply:
I am not sure what you mean. The snap (aka the svg image of the ggplot) is created from the code and the unit test will always correspond to the code....

Reply:
It should be independent. Perhaps we can organize a spring where we implement this. For now OK.